src/redux/saga/employer.ts

```diff
@@ -30,7 +30,7 @@ function* registerEmployerSaga() {
 function* loginEmployerSaga() {
   try {
     const credentialData = yield select(credential)
-    const res = yield axios.post('/employer/login/local', {
+    const res = yield axios.post('/login/local', {
```

Comment:
Since now login for `jobseeker` and `employer` are the same. Can you make only one action for login: `loginUserSaga`